views/index.hbs

```diff
@@ -1,6 +1,6 @@
 ﻿<h1 class="f1 f-6-ns orange tc"><a class="orange no-underline" href="/">Hacktoberfest Checker</a></h1>
 
-<div class="tc">
+<div class="tc username-form">
```

Comment:
Can we use a tachyone class rather than define a new one? See http://tachyons.io/docs/layout/spacing/

Reply:
Ahh, sure, will update